docs/deploystack.md

```diff
@@ -0,0 +1,10 @@
+## Install
+You can install this application using the `Open in Google Cloud Shell` button 
+below. 
+<!-- TODO: remove reference to the deploystack-enable branch when it pushes to main -->
```

Comment:
**Reminder:**
Thanks for adding this `TODO` comment as a reminder.
Just to make sure _I_ don't forget before merging, I'm leaving this comment here. 🎗️